README.md

```diff
@@ -17,9 +17,9 @@ e.g)/hoge/pre_image
 2. tr_image_movie.py
     * converts a series of `.tif`s into `.mp4` in a designated order.
```

Comment:
converts a series of `.tif`s into `.mp4` in a designated order (time-continuous or depth-continuous).